Oleksii_Makarov/HW6/Lesson6_1.py

```diff
@@ -0,0 +1,15 @@
+from random import randint
+import os
+
+rand_tuples = list()
+
+for temp in range(0, 100):
+    right = randint(0, 100)
+    left = randint(0, 100)
+    operation = randint(1, 3)
+    rand_tuples.append((right, left, operation))
+
+os.makedirs('test/data')
```

Comment:
what if I want to run your program for the second time?

Reply:
> what if I want to run your program for the second time?

Added if for os.makedirs